util/setuidgid.c

```diff
@@ -126,7 +126,10 @@ set_group(const char * groupname)
 	/* Find numerical GID. */
 	errno = 0;
 	if ((group_info = getgrnam(groupname)) == NULL) {
-		warnp("getgrnam(\"%s\")", groupname);
+		if (errno)
+			warnp("getgrnam(\"%s\")", groupname);
+		else
+			warn0("No such groupname: %s", groupname);
```

Comment:
I'd say `no such group` (and below, `no such user`) rather than group *name*?

Or at very least add a space before "name".